sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -289,7 +289,7 @@ public Builder setCapabilities(Capabilities capabilities) {
          * @throws IllegalStateException if config validation failed when updating the target config for a default
          *     {@link OfferRequirementProvider}, or if creating a default {@link ConfigStore} failed
          */
-        public DefaultScheduler build() {
+        public DefaultScheduler build() throws ConfigStoreException {
```

Comment:
If it must be so, but I think I'd be a little surprised to catch a ConfigStoreException from this method in general.